app/Http/Controllers/Rpc/WalletController.php

```diff
@@ -137,6 +140,38 @@ public function depositInfo()
         return self::json($resp);
     }
 
+    public function history()
+    {
+        $userId = Auth::user()->id;
+        $resp = [];
+        //TODO pagination
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)

Reply:
I will do it.